packages/nodes-base/nodes/Google/Gmail/v2/MessageDescription.ts

```diff
@@ -234,6 +234,83 @@ export const messageFields: INodeProperties[] = [
 			},
 		],
 	},
+	{
+		displayName: 'Options',
+		name: 'options',
+		type: 'collection',
+		placeholder: 'Add Option',
+		displayOptions: {
+			show: {
+				resource: ['message'],
+				operation: ['send'],
+			},
+		},
+		default: {},
+		options: [
+			{
+				displayName: 'Attachments',
+				name: 'attachmentsUi',
+				placeholder: 'Add Attachment',
+				type: 'fixedCollection',
+				typeOptions: {
+					multipleValues: true,
+				},
+				options: [
+					{
+						name: 'attachmentsBinary',
+						displayName: 'Attachment Binary',
+						values: [
+							{
+								displayName: 'Attachment Field Name',
+								name: 'property',
+								type: 'string',
+								default: 'data',
+								description:
+									'Add the field name from the input node. Multiple properties can be set separated by comma.',
+								hint: 'The name of the field with the attachment in the node input',
+							},
+						],
+					},
+				],
+				default: {},
+				description: 'Array of supported attachments to add to the message',
+			},
+			{
+				displayName: 'BCC',
+				name: 'bccList',
+				type: 'string',
+				description:
+					'The email addresses of the blind copy recipients. Multiple addresses can be separated by a comma. e.g. jay@getsby.com, jon@smith.com.',
+				placeholder: 'info@example.com',
+				default: '',
+			},
+			{
+				displayName: 'CC',
+				name: 'ccList',
+				type: 'string',
+				description:
+					'The email addresses of the copy recipients. Multiple addresses can be separated by a comma. e.g. jay@getsby.com, jon@smith.com.',
+				placeholder: 'info@example.com',
+				default: '',
+			},
+			{
+				displayName: 'Sender Name',
+				name: 'senderName',
+				type: 'string',
+				placeholder: 'e.g. Nathan',
+				default: '',
+				description: "The name that will be shown in recipients' inboxes",
+			},
+			{
+				displayName: 'Reply To Email',
+				name: 'replyTo',
+				type: 'string',
+				placeholder: 'reply@example.com',
+				default: '',
+				description: 'The email address that the reply message is sent to',
```

Comment:
no need to duplicate all options, you can display option conditionally to operation
```
		displayOptions: {
			show: {
				'/operation': ['send'],
			},
		},
		displayOptions: {
			hide: {
				'/operation': ['reply'],
			},
		},
```